### ISSUE TITLE
Crash report for v3.3.13082 64 bit pre-release

### ISSUE THREAD

**Issue by kyeongsoo** [sumatrapdfreader/sumatrapdf#1709]
I am submit a [crash report](https://gist.github.com/kyeongsoo/0f9bf4cb8de079585287e773b9b65aef#file-gistfile1-txt).

**Comment by kjk** [comment]
Crash:
```
Faulting IP: 00007FF6CE559CE1 01:0000000000128CE1 sumatrapdf.exe!TabsOnNotify+0xe1 D:\a\sumatrapdf\sumatrapdf\src\Tabs.cpp+858
Fault writing address 0000000000000000

00007FF6CE559CE1 01:0000000000128CE1 sumatrapdf.exe!TabsOnNotify+0xe1 D:\a\sumatrapdf\sumatrapdf\src\Tabs.cpp+858
00007FF6CE558865 01:0000000000127865 sumatrapdf.exe!TabNotification+0x75 D:\a\sumatrapdf\sumatrapdf\src\Tabs.cpp+394
00007FF6CE57E090 01:000000000014D090 sumatrapdf.exe!uitask::WndProcTaskDispatch+0x20 D:\a\sumatrapdf\sumatrapdf\src\utils\UITask.cpp+19
```

Looks like something related to tab management when reloading a document but can't make the sense of it. The crash implies forced `CrashIf()` but I don't see any in that locations (maybe inlined?).

Also, why so many `StartMonitoringDirForChangesAPC` calls for the same directory? Are those reloads?

**Comment by kyeongsoo** [comment]
Sorry for the late response; those calls may be related with either manual reloading or automatic one (of PDF files) after latex compilation. Anyhow, now I am using Ver: 3.3.13112 and haven't had such issues.

**Comment by GitHubRulesOK** [comment]
@kyeongsoo Thanks for the feedback its appreciated
@kjk 
there were a couple of fixes in that period such as those for Crash report for v3.3.13104 64 bit pre-release #1729
and the related fix mismatched gAllowAllocFailure accounting (fixes #1693) (3.3.13063+)

Even if those were not the cure, since this is reported as resolved by @kyeongsoo  using 3.3.13112 am closing 
Can be reopened if a similar event happens with more recent versions
